### PR TITLE
Convert seconds to miliseconds before adding them to retry timeout

### PR DESCRIPTION
**Pull request by VVeremjova**
@okozlovlux, @LevchenkoS, @LitvinenkoIra please review

### CODE REVIEW

File: src/components/policy/policy_regular/src/policy_manager_impl.cc [hunk 1]
```diff
@@ -314,7 +314,8 @@ void PolicyManagerImpl::StartPTExchange() {
       if (RequestPTUpdate() && !timer_retry_sequence_.is_running()) {
         // Start retry sequency
         const int timeout_sec = NextRetryTimeout();
-        LOG4CXX_DEBUG(logger_, "Start retry sequence timeout = " << timeout_sec);
+        LOG4CXX_DEBUG(logger_,
+                      "Start retry sequence timeout = " << timeout_sec);
```
Comment (jacobkeeler) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: This change is irrelevant (fixes a formatting issue) and already exists in develop.

File: src/components/policy/policy_regular/src/policy_manager_impl.cc [hunk 2]
```diff
@@ -778,7 +779,7 @@ uint32_t PolicyManagerImpl::NextRetryTimeout() {
   ++retry_sequence_index_;
 
   for (uint32_t i = 0u; i < retry_sequence_index_; ++i) {
-    next += retry_sequence_seconds_[i];
+    next += retry_sequence_seconds_[i] * 1000;
     // According to requirement APPLINK-18244
     next += retry_sequence_timeout_;
```
Comment (jacobkeeler) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: Why is the default timeout added in this for loop rather than outside??

**Comment by VVeremjova** [comment]
@jacobkeeler Please use fixed script (commit [4f4386c0a8](https://github.com/smartdevicelink/sdl_atf_test_scripts/commit/4f4386c0a87e1227f00fcfad3b7df9151f54cc61) ) -  message `expected: 30000msec` was in non-fixed script.

File: src/components/policy/policy_regular/src/policy_manager_impl.cc (hunk 2 above)
Comment (VVeremjova) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @jacobkeeler According to requirement 

>The timeout of the N retry must be count the following way:   time for_the_ (N)retry = time_ for_ the_(N-1)retry + <"seconds_between_retries">[N] + timeout
where N>=1 and retries are started to be numerated from "0"

File: src/components/policy/policy_regular/src/policy_manager_impl.cc (hunk 1 above)
Comment (VVeremjova) [inline src/components/policy/policy_regular/src/policy_manager_impl.cc]: @jacobkeeler Rebased on develop - see edd1b2b

**Comment by jacobkeeler** [comment]
My ATF branch was indeed outdated, updating allowed the test to pass, so I approve

**Comment by VVeremjova** [comment]
Rebuild required

**Pull request merged**